modules/installation/pages/install-server-unified.adoc

```diff
@@ -8,6 +8,12 @@ For this topic we assume that you already have valid organization credentials wi
 
 For information on registering with {scc}, retrieving your organization credentials from {scc}, or obtaining installation media, see xref:general-requirements.adoc[].
 
+[NOTE]
+====
+The recommended way of installing {susemgr} is from the {sle} installation media and the Unified Installer.
+In case of installing {susemgr} in a public cloud where {susemgr} image is not available, it is possible to change a {sles}{nbsp}{sles-version}{nbsp}{sp-version} into a {susemgr}{nbsp}{productnumber}.
+For steps how to do that follow the instructions at installation:install-vm.adoc[].
```

Comment:
I am not sure at all about this note.

Reality is:
- Use always the Unified installer unless:
  - You are installing on a cloud where SUSE Manager images are available
  - You are want to install on an VM using JeOS

If the public cloud does not have a SUSE Manager image, I am not sure if what you described will work, or if it's supported at all even if it works.

Keep in mind the link you provide mentions JeOS. I am not sure if the SLE images on public clouds are based on JeOS.

I guess we need to get confirmation from @paususe for this.

Reply:
I would link to the Public Cloud Quick Start, or the pubcloud pages in the Install Guide. Also, remember the "for more ... see" formulation for xrefs.